### ISSUE TITLE
Change help icon to info

### ISSUE THREAD

**Issue by pihart** [cjquines/qboard#143]
It looks more appealing to me, but more importantly it conveys intent well
![image](https://user-images.githubusercontent.com/18153122/119437362-d5241d80-bcd2-11eb-9b06-5f32feee370a.png)

**Comment by cjquines** [comment]
but still keep it named `help`? hm. i feel like if youre making it info we should rename it to info

**Comment by pihart** [comment]
Good point. On a personal level, I'm good either way; the issue is mainly because it's certainly *not* a "question."

**Comment by cjquines** [comment]
"info" does make more sense, i think, compared to "help". but im still not sure what the best solution is. honestly its more like a splash screen or a welcome modal or something idk

**Comment by pihart** [comment]
Maybe the `rocket` or `play` icons with the name "Start"?

**Comment by cjquines** [comment]
"bindings" makes most sense, i think, in light of #158 wanting a different settings modal?
